### PR TITLE
cleaner funded issue urls

### PR DESCRIPTION
**Pull request by owocki**
before: https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/450
after: https://gitcoin.co/issue/gitcoinco/web/450
backwards compatible: ✅

### CODE REVIEW

**Comment by owocki** [comment]
i _could_ migrate all of the kill/submit/accept functionality.. but figured thatll be easier to do once https://github.com/gitcoinco/web/issues/406 is done

**Comment by owocki** [comment]
@mbeacom just resolved merge conflicts and isort issues.. can i get an 👀 ?

File: app/assets/v2/js/pages/dashboard.js [hunk 1]
```diff
@@ -187,11 +187,7 @@ var refreshBounties = function(){
                 } else if(result['fulfiller_address'] != '0x0000000000000000000000000000000000000000'){
                     result['my_bounty'] = '<a class="btn font-smaller-2 btn-sm btn-outline-dark" role="button" href="#">'+result['status']+'</span></a>';
                 }
-                if (result['web3_type'] == 'legacy_gitcoin') {
-                    result.action = '/legacy/funding/details?url=' + result.github_url;
```
Comment (mbeacom) [inline app/assets/v2/js/pages/dashboard.js]: Will this 💥 legacy bounty rendering?

Comment (owocki) [reply app/assets/v2/js/pages/dashboard.js]: just put in a fix for htis

**Comment by owocki** [comment]
27011114bdeb3ae033b0a28692d224abaf2e32e5 fixes the tests

**Pull request merged**